### ISSUE TITLE
Multiple classification

### ISSUE THREAD

**Issue by leesh9069** [leesh9069/growth_recording#1]
### Multiple Classification
<hr>

-  여러 개의 이미지를 분류하는 과정에서 확률의 합이 1이 되지 않으면 정확한 모델 신뢰도 측정이 불가하다.
-  Softmax 를 이용해 확률의 합을 1로 정규화하여야 한다.
- 출력층을 통과한 값들이 Softmax를 거치며 적절한 확률값으로 변환된다.

###  Cross Entropy
<hr>

- 다중 분류 모델의 Loss function
- Cross Entropy 를 편미분하여 파라미터 값을 조정해나간다.
